src/lib/util/webhook.ts

```diff
@@ -90,11 +82,29 @@ export async function sendToChannelID(
 				}
 			}
 		} else {
-			await channel.send({
-				content: data.content,
-				files,
-				embeds
-			});
+			const message = await channel.send(data);
+			if (user) {
+				try {
+					const selection = await message.awaitMessageComponentInteraction({
+						time: Time.Minute * 10,
+						filter: i => {
+							if (i.user.id !== user.id) {
+								i.reply({ ephemeral: true, content: 'This is not your confirmation message.' });
+								return false;
+							}
+							return true;
+						}
+					});
+					if (componentSelection && componentSelection[selection.customID]) {
+						message.author = user;
+						componentSelection[selection.customID](message);
+					}
+					await message.edit({ components: [] });
+				} catch (e) {
+					await message.edit({ components: [] });
+				} finally {
+				}
```

Comment:
empty finally, maybe you meant to put the edit in here - (but still need a catch)